pkg/private/build_tar.py

```diff
@@ -19,10 +19,10 @@
 import tarfile
 import tempfile
 
-import archive
-import helpers
-import build_info
-import manifest
+from private import archive
+from private import helpers
+from private import build_info
+from private import manifest
```

Comment:
This is to make it easier for tools to figure out where paths are, given WORKSPACE-relative paths, right?

I'm theoretically fine with this, but it would seem odd to people who aren't used to this sort of scheme, myself included.  Consider documenting it somewhere.

Reply:
Yes. It is mostly for people who import the code into their WORKSPACE. They might land it at `/third_party/bazel/rules_pkg` and want to normalize the paths so their Python code only has a single import path at the top.

I'm torn between a local guide to this or the overall "blaze ecosystem" style guide. I have an AI to take a crack at the latter.

Reply:
Tracked in #412